### ISSUE TITLE
Moon superimposed

### ISSUE THREAD

**Issue by JoeyMengDi** [amyznikov/SerStacker#1]
HI，Andrey Myznikov
       Have you tried the superposition of the moon surface, I don’t know how it works

**Comment by amyznikov** [comment]
Hello JoeyMengDi,
 Not sure if I understand what you mean the 'superposition of the moon surface', may be you mean the creating a 'moisaic'..
If so - I have not implemented it yet, but I am planning to implement it too, but later.

Currently I need to finish the  Jupiter derotation algorithm, after that I probably will update the GUI (it is very crazy now), and then create mosaics from moon images.

**Comment by JoeyMengDi** [comment]
Yes, you should have used AutoStakkert software, similar to this function, look forward to your new features below
